quizbizz/src/util/api/fetchApi.js

```diff
@@ -0,0 +1,31 @@
+import Axios from "axios";
+
+const fetchApi = async (url, body, isAuthRequired) => {
+  const token = localStorage.getItem("token");
+  try {
+    const response = await Axios.get(
+      url,
+      { body },
+      {
+        headers: {
+          authorization: isAuthRequired ? token : "",
+        },
+      }
+    );
+
+    return {
+      data: response.data,
+      success: true,
```

Comment:
why there is no returning status code here, in the postApi.js you are returning the status code also.

Reply:
Nice catch